code/game/objects/structures/crates_lockers/closets/secure/personal.dm

```diff
@@ -4,6 +4,18 @@
 	req_access = list(ACCESS_ALL_PERSONAL_LOCKERS)
 	var/registered_name = null
 
+/obj/structure/closet/secure_closet/personal/examine(mob/user)

```

Comment:
As with genpop, the personal lockers should be made personal purely because of their electronics, not any special poperties of the metal used to build them. Maybe add a small menu in the airlock electronics that would let you pick what you want to make?